cl/citations/description_utility.py

```diff
@@ -0,0 +1,50 @@
+import re
+from math import log
+
+from cl.search.models import OpinionCluster
+
+_GERUND = r"(?:\S+ing)"
+_GERUND_THAT = rf"{_GERUND} that"
+_HOLDING = r"(?:holding|observing|deciding|ruling|recognizing)"
+_HOLDING_THAT = rf"{_HOLDING} that"
+
+OPTIMAL_NUM_WORDS = 20
+ALLOWED_NUM_WORDS_DEVIATION = 5
```

Comment:
Where did these values come from? Perhaps a comment explaining how these were derived for poor, sad people in the future?